resources-service/tests/service/server.ts

```diff
@@ -39,6 +39,10 @@ export const defaultConfig: {
 export const getServer = (config?: Partial<typeof defaultConfig>) => {
   process.env.AWS_ACCESS_KEY_ID = 'mock-access-key';
   process.env.AWS_SECRET_ACCESS_KEY = 'mock-secret-key';
+  process.env.ELASTICSEARCH_CONFIG = JSON.stringify({
```

Comment:
I assume we are spinning up a real ElasticSearch instance at some point, struggling to see where it gets run, is it in a different PR?

How is the start up time & test speed using a real ElasticSearch instance? (I rely on service tests heavily to do HRM development so have a keen interest in keeping them spritely :-P)